<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kuechewindow</class>
 <widget class="QDialog" name="kuechewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0.0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(50, 50, 50, 255));</string>
  </property>
  <widget class="QPushButton" name="pushButtonLightSpeis">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>115</width>
     <height>78</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Speis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLight">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>4</y>
     <width>115</width>
     <height>78</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Hänge-
Lampe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightWand">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>86</y>
     <width>115</width>
     <height>78</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Wand</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLightAbwasch">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>86</y>
     <width>115</width>
     <height>78</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Abwasch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>361</x>
     <y>168</y>
     <width>115</width>
     <height>78</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Zurück</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonBack</sender>
   <signal>clicked()</signal>
   <receiver>kuechewindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
